src/VkBootstrap.cpp

```diff
@@ -1774,10 +1774,23 @@ detail::Result<Swapchain> SwapchainBuilder::build() const {
 		return detail::Error{ SwapchainError::failed_query_surface_support_details, surface_support_ret.vk_result() };
 	auto surface_support = surface_support_ret.value();
 
-	uint32_t image_count = surface_support.capabilities.minImageCount + 1;
+	uint32_t image_count;
```

Comment:
I generally prefer defaulting all variables unless absolutely necessary that they be uninitialized.
```suggestion
	uint32_t image_count = info.min_image_count;
```